server/services/store/sqlstore/migrate.go

```diff
@@ -65,6 +65,7 @@ func (s *SQLStore) getMigrationConnection() (*sql.DB, error) {
 	if s.configFn != nil {
 		settings = s.configFn().SqlSettings
 	}
+	*settings.DriverName = s.dbType
```

Comment:
Hmm .. what was the issue here? Would there be any case where they would be different?

Reply:
Normally doesn't but it's not being set for standalone version. Hence we use the default values but default value for driver is psql.

Reply:
Then we need to set the datasource as well right?

Reply:
Hmm, yeah but it's not being used in the `SetupConnection` function, but for consistency, we can do that.

Reply:
Oh actually, it is being passed via `s.connectionString`. Okay then no need.